server/seed/cassandra/generateProductCSV.js

```diff
@@ -0,0 +1,38 @@
+const models = require('../../models/postgres/index.js');
+const csvWriter = require('csv-write-stream');
+const writer = csvWriter();
+const faker = require('faker');
+const fs = require('fs');
+
+const random = (min, max) => {
+  return Math.floor(Math.random()*(max-min) + min);
+}
+
+(() => {
+  writer.pipe(fs.createWriteStream(__dirname + '/product_cassandra.csv'));
+  let i=10000000;
+
+  function writeProducts() {
+    var ok = true;
+    do {
+      i -= 1;
+      if (i === 0) {
+        // last time!
+        writer.write({id: 10000000-i, product_item: faker.commerce.productName(), liked: false});
```

Comment:
Wow, that's cool. Good idea with the id property, removing the current value of i from the max.